lib/hamster/core_ext/struct.rb

```diff
@@ -0,0 +1,9 @@
+class Struct
+  # Implement Struct#to_h for Ruby interpreters which don't have it
+  # (such as MRI 1.9.3 and lower)
+  unless method_defined?(:to_h)
+    def to_h
+      Hash[members.zip(values)]
```

Comment:
Maybe simpler/clearer:

    Hash[each_pair.to_a]

(not that it's really important...)